fluent-plugin-google-cloud.gemspec

```diff
@@ -19,6 +19,7 @@ eos
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
+  gem.add_runtime_dependency 'chronic_duration', '~> 0.10.6'
```

Comment:
Per discussion offline, we decided to use simple regex instead of this third-party lib since it is not actively maintained. I just modified the implementation.